### PR TITLE
Features/doc update

### PR DESCRIPTION
**Pull request by Aboudjem**

### CODE REVIEW

**Review by zakrad — CHANGES_REQUESTED** [review]

**Review by mohamadhammoud — COMMENTED** [review]
I do think that it needs spaces, I didn't see any concatenated topic before.

File: README.md [hunk 1]
```diff
@@ -1,75 +1,132 @@
-## Scripts
+# DLT (DualLayerToken) Standard 🛠️
 
-This repository includes the following scripts in the `package.json` file:
+Welcome to the DualLayerToken (DLT) Standard, proposed under EIP-6960! 🎉
```
Comment (mohamadhammoud) [inline README.md:3]: **DualLayerToken** I didn't see any concatenated topics before, does it need spaces ?

File: README.md [hunk 2]
```diff
@@ -1,75 +1,132 @@
...
 
-This command runs the Prettier and Solhint linters on all the Solidity files in the project and automatically fix any issues it finds.
+- `setApprovalForAll(operator, approved)`: Allows the approval or removal of `operator` as an operator for the caller. Operators can call `transferFrom` or `safeTransferFrom` for any token owned by the caller.
+- `safeTransferFrom(sender, recipient, mainId, subId, amount, data)`: Moves `amount` tokens from `sender` to `recipient` using the allowance mechanism. `amount` is then deducted from the caller's allowance.
+- `approve(spender, mainId, subId, amount)`: Sets `amount` as the allowance of `spender` over the caller's tokens.
+- `subBalanceOf(account, mainId, subId)`: Returns the amount of tokens owned by `account
+- `balanceOfBatch(accounts, mainIds, subIds)`: Returns the balances of multiple `accounts` for each pair of `mainIds` and `subIds`.
+- `allowance(owner, spender, mainId, subId)`: Returns the remaining number of tokens that `spender` can spend on behalf of `owner` for a specific `mainId` and `subId`.
+- `isApprovedForAll(owner, operator)`: Checks if `operator` is allowed to manage all of the assets of `owner`.
```
Comment (mohamadhammoud) [inline README.md:71]: Are we coloring the methods and the parameters ? or we are coloring the normal words.

Methods in blue and others in black

Comment (Aboudjem) [reply README.md:71]: Screenshot 2023-05-16 at 7 51 07 PM
it's an issue on the visualizer, only the functions variables and functions names are highlited

**Pull request merged**